lib/src/configuration/better_player_controls_configuration.dart

```diff
@@ -206,15 +207,15 @@ class BetterPlayerControlsConfiguration {
       this.enablePip = true,
       this.enableRetry = true,
       this.overflowMenuCustomItems = const [],
-      this.overflowMenuIcon = Icons.more_vert,
-      this.pipMenuIcon = Icons.picture_in_picture,
-      this.playbackSpeedIcon = Icons.shutter_speed,
-      this.qualitiesIcon = Icons.hd,
-      this.subtitlesIcon = Icons.text_fields,
-      this.audioTracksIcon = Icons.audiotrack,
+      this.overflowMenuIcon = Icons.more_vert_outlined,
+      this.pipMenuIcon = Icons.picture_in_picture_outlined,
+      this.playbackSpeedIcon = Icons.shutter_speed_outlined,
+      this.qualitiesIcon = Icons.hd_outlined,
+      this.subtitlesIcon = Icons.closed_caption_outlined,
+      this.audioTracksIcon = Icons.audiotrack_outlined,
       this.overflowMenuIconsColor = Colors.black,
-      this.forwardSkipTimeInMilliseconds = 15000,
-      this.backwardSkipTimeInMilliseconds = 15000,
+      this.forwardSkipTimeInMilliseconds = 10000,
+      this.backwardSkipTimeInMilliseconds = 10000,
```

Comment:
This is also changes unrelated to UI.

Reply:
This has actually changed based on the ui change. The default icon for skipping forward or back says your skipping at a rate of 10 seconds, this however is still modifiable when constructing the player controls configurations instance.